src/GenioApp.cpp

```diff
@@ -461,6 +476,301 @@ GenioApp::_PrepareConfig(ConfigManager& cfg)
 }
 
 
+// Genio Scripting
```

Comment:
Can we evaluate to move scripting related objects to another file?

Reply:
I tried but reverted it back as I got too many errors. I plan to try again, though.